lib/truffle/java/README.md

```diff
@@ -0,0 +1,89 @@
+# JRuby-compatible Java interop
+JRuby provides a rich set of interop functionality between Ruby and
+Java, and we attempt to emulate it faithfully in TruffleRuby.
```

Comment:
That's a very dense Markdown file :smile: 
Usually we have some blank lines around titles, etc, but as long as it renders well it doesn't matter.